web-client/src/js/view/homePage/CreateTestPanel.js

```diff
@@ -0,0 +1,40 @@
+import React, { Component } from 'react';
+import ReactDOM from 'react-dom';
+import { Row, Col, div, Button} from 'react-bootstrap';
+import GridHeader from '../../gridHeader/GridHeader'
+import TestDefinition from '../testDefinition/TestDefinition'
+
+class CreateTestPanel extends Component {
+
+  createTest() {
+    ReactDOM.render(<TestDefinition />, document.getElementById('root'));
+  }
+
+  render() {
+    return (
+      <div className='panel panel-default'>
+        <Row>
+          <Col md={12}>
+            <GridHeader />
+            <h4 fluid style={{ paddingLeft: 15 }}>Documentation:</h4>
+            <h1>{}</h1>
+            <Row>
+              <Col md={3}></Col>
+              <Col md={6}>
+                <div>
+                  <Button onClick={this.createTest} bsStyle="primary" bsSize="large" block>
```

Comment:
Should we not create a common button component, which will encapsulate the styling and will be consistent through out the project.